core/contracts/common/implementation/MultiRole.sol

```diff
@@ -11,7 +11,7 @@ library Exclusive {
     }
 
     function resetMember(RoleMembership storage roleMembership, address newMember) internal {
-        require(newMember != address(0x0), "Cannot set an exclusive role to 0x0");
+        require(newMember != address(0x0));
```

Comment:
Did we need to *remove* this pre-existing require message? I don't think this one breaks the non optimized compilation.

Reply:
I removed it to be consistant with the other require messages on these functions